auth/src/main/res/values/styles.xml

```diff
@@ -3,9 +3,9 @@
            tools:ignore="ResourceName">
 
     <style name="FirebaseUI" parent="Theme.AppCompat.DayNight.DarkActionBar">
-        <item name="colorPrimary">@color/fui_colorPrimary</item>
-        <item name="colorPrimaryDark">@color/fui_colorPrimaryDark</item>
-        <item name="colorAccent">@color/fui_colorAccent</item>
+        <item name="colorPrimary">@color/colorPrimary</item>
+        <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
+        <item name="colorAccent">@color/colorAccent</item>
```

Comment:
@samtstern The reason we defined those colors was so that we could use them here. However, we are expecting them to be overridden so we don't actually need them to be fully defined colors, but my knowledge of the Android resource system wasn't that great back then. Anyway, now we just define them as ids of type `color` so the build will properly fail if devs using FirebaseUI don't define their colors and they'll be properly overridden if they are defined.